### PR TITLE
Added forced truncation of WKT types in Markdown/HTML rendering.

### LINKED ISSUE #407: rf_ipython custom format for geometry objects in spark DataFrame
Related to  #383, basically fix the same condition for the markdown and html renderers of Spark DataFrames. The fix for #383 only corrected pandas DF.

### PR DESCRIPTION
**Pull request by metasim**
Closes #407

### CODE REVIEW

File: core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala [hunk 1]
```diff
@@ -47,8 +49,9 @@ trait DataFrameRenderers {
               lit("\"></img>")
```
Comment (metasim) [inline core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala]: Wondering if the line above this one
```
 base64(rf_render_png(rf_resample(resolved, 0.5), ColorRamps.Viridis))
```
should be changed to use `ColorRamps.greyscale(256)`, and then apply a color ramp in the python API for notebooks.

Comment (vpipkt) [reply core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala]: I guess I am not sure about what we can do on the notebook / python side if the image is already b64 encoded in HTML... Some css stuff? idk

**Comment by vpipkt** [comment]
🚢 it

File: core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala (hunk 1 above)
Comment (metasim) [reply core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala]: Good point. Damn. Didn't think of that.

**Pull request merged**